airflow/settings.py

```diff
@@ -310,3 +310,12 @@ def initialize():
 
 # Used by DAG context_managers
 CONTEXT_MANAGER_DAG = None
+
+# If dagcached is enabled, scheduler writes serialized DAGs to DB, and webserevr
+# reads DAGs from DB instead of importing from files.
+DAGCACHED_ENABLED = conf.getboolean('core', 'dagcached', fallback=True)
```

Comment:
The default behavior should be still parsing from files until 2.0

Changing it to the following

```suggestion
DAGCACHED_ENABLED = conf.getboolean('core', 'dagcached', fallback=False)
```

Reply:
I turned it on for testing and forgot to set it back.. Thanks for fixing it